code-samples/sample.json

```diff
@@ -0,0 +1,31 @@
+{
+    "deep 1": "test",
+    "Sequence": ["item1", "item2"],
+    "SequenceWithValues": 
+    [
+        {
+            "key": "value",
+            "key2": "value2",
+        }
+    ],
+    "glosrsay": {
```

Comment:
Is this guy a typo?

Reply:
Yep, it's a typo, could be "Glossary". But it's just a sample file used to test the colors. It has no impact on the theme.

Reply:
Cool, thanks for clarifying :100: